docs/readthedocs/source/doc/Nano/QuickStart/index.md

```diff
@@ -48,9 +48,7 @@
 
 ---------------------------
 
-
-- [**BigDL-Nano Hyperparameter Tuning (Tensorflow Sequential/Functional API) Quickstart**](./pytorch_quantization_openvino.html)
-
+- [**BigDL-Nano Hyperparameter Tuning (Tensorflow Sequential/Functional API) Quickstart**](../Tutorials/seq_and_func.html)
```

Comment:
Not sure whether `docs/readthedocs/source/doc/Nano/Tutorials/` a good place to put these ipynb files, and whether we want to copy the whole file here.

I tried to directly link to the ipynb file under the python folder, but it does not work